content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -434,7 +434,7 @@ node_templates:
 This relationship job is to inject helm environment variables locations to release/repo nodes.
 Target node is cloudify.nodes.helm.Binary which creates temporary environment for each binary.
 
-This relationship will derived from  cloudify.relationships.connected_to relationship.   
+This relationship will derived from cloudify.relationships.connected_to relationship.   
```

Comment:
`The relationship is derived from the cloudify.relationships.connected_to relationship type.`